### ISSUE TITLE
Investigate support for managed identity

### ISSUE THREAD

**Issue by weidongxu-microsoft** [Azure/azure-libraries-for-java#1343]
https://microsoft.sharepoint.com/teams/ep/SitePages/All-Offerings-must-support-Managed-Service-Identities-(MSI).aspx

Basically all services that runs code need to support it.

**Comment by blacelle** [comment]
Just to confirm, is-this a pre-requisite to enable https://docs.microsoft.com/en-us/azure/aks/use-managed-identity#create-an-aks-cluster-with-managed-identities through this library?

**Comment by weidongxu-microsoft** [comment]
No, managed identity can be enabled via any approach available, Azure CLI, powershell, Portal, SDK, etc.

**Comment by ChenTanyi** [comment]
@blacelle If you want to enable managed identity for AKS through Java SDK, you can see my sample https://github.com/tanyi-test/azure-k8s-pod-with-identity
